xslts/distributions-add-function.md

```diff
@@ -5,13 +5,23 @@ Ajoute un élément de type `gmd:function` à une distribution.
 
 ## Paramètres
 
+<div class="fr-table">
+<div class="fr-table__wrapper">
+<div class="fr-table__container">
+<div class="fr-table__content">
```

Comment:
```suggestion
<div class="fr-table"><div class="fr-table__wrapper"><div class="fr-table__container"><div class="fr-table__content">
```

Reply:
Might be more maintainable since we can't ident? Not sure.

(same for closing, obv)